CrowServe/CrowServe.h

```diff
@@ -94,8 +144,9 @@ class Socket {
     
 private:
     u32 m_socket = 0;
+    std::jthread m_thread;
+    CrownLink::CrownLinkProtocol m_crownlink_protocol;
+    P2P::P2PProtocol m_p2p_protocol;
```

Comment:
Not important, should probably renames these to just Protocol, since they are already namespaced, and so it creates a redundancy here.